### ISSUE TITLE
no way to assert OK outcome from a userDelete request in 1.8.x and 2.0.x

### ISSUE THREAD

**Issue by davidkarlsen** [seratch/jslack#207]
The userDeleteResponse used to have properties to check for a OK outcome:
```
Assert.isTrue(
        slackApiResponse.isOk(),
        String.format(
            "Error from slack API: %s %s",
            slackApiResponse.getError(), slackApiResponse.getWarning()));

    return slackApiResponse;
```

The updated api does not have any attributes to query for outcome. Does this mean it will be checked by the API (e.g. userDelete() will raise an exception if some problem?)

**Comment by seratch** [comment]
Sorry for the confusion. I'm ashamed to say that the previous implementation was totally wrong and that's the reason I decided to break backward compatibilities. Actually, the flag we had before doesn't work as you expect. The only reason we had it was that we just inherited the same interface with other APIs.

It depends on SCIM API's behavior but we have no ways to check if the deletion succeeded or not.  The API always returns 200 OK with empty body as of today.
https://github.com/seratch/jslack/commit/9b24fb9eb1759f1bbf5b8384bae52d06eac78038
